browser/resources/settings/email_aliases_page/content/email_aliases_list.tsx

```diff
@@ -0,0 +1,186 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import * as React from 'react'
+import { Alias, MappingService, MAX_ALIASES, ViewState } from './types'
+import { Card, Col, Row } from './style'
+import styled from 'styled-components'
+import Icon from '@brave/leo/react/icon'
+import { color } from '@brave/leo/tokens/css/variables'
+import Tooltip from '@brave/leo/react/tooltip'
+import { getLocale } from '$web-common/locale'
+import formatMessage from '$web-common/formatMessage'
+import Button from '@brave/leo/react/button'
+import ButtonMenu from '@brave/leo/react/buttonMenu'
+
+const AliasLeoMenuItem = styled('leo-menu-item')`
+  & ${Row} { font-size: 90%; }
+  & ${Row} span { margin-inline-start: 0.5em; }
+`
+
+const AliasItemRow = styled(Row)`
+  margin: 0px;
+  font-size: 125%;
+  padding: 18px 0px 18px 25px;
+  border-top: ${color.legacy.divider1} 1px solid;
```

Comment:
hey, is this the right token? I think we probably want to avoid using legacy tokens

Reply:
fixed